src/main/java/areas/Oceanarium.java

```diff
@@ -0,0 +1,5 @@
+package areas;
+
+public interface Oceanarium {
+    void doILiveInOceanarium();
```

Comment:
it's not clear what this method supposed to do, i think swimmable is enough

Reply:
fixed